rocky/katalogus/views/boefje_setup.py

```diff
@@ -232,3 +254,22 @@ def create_boefje_with_form_data(form_data, plugin_id: str, created: str | None)
         oci_image=form_data["oci_image"],
         oci_arguments=arguments,
     )
+
+
+def get_interval_minutes(interval_number, interval_frequency) -> int:
+    interval_minutes = 0
+    if interval_number == 0:
+        return None
+
+    if interval_frequency == "minutes":
+        interval_minutes = interval_number
```

Comment:
You could simply return the int()ed result immediately. No need for all the elif branches.